src/argus/notificationprofile/signals.py

```diff
@@ -1,12 +1,47 @@
+from django.db.utils import ProgrammingError
+
+from rest_framework.exceptions import APIException
+
 from argus.auth.models import User
 from .models import DestinationConfig, TimeRecurrence, Timeslot
 
 __all__ = [
+    "sync_media",
     "create_default_timeslot",
     "sync_email_destination",
 ]
 
 
+def sync_media(sender, **kwargs):
+    """Sync MEDIA_PLUGINS with the Media table
+
+    Check if all media in Media has a respective class"""
+
+    from .media import MEDIA_CLASSES, MEDIA_CLASSES_DICT
+
+    apps = kwargs['apps']
+    try:
+        Media = apps.get_model('argus_notificationprofile', 'Media')
+    except ImportError:
+        return
+
+    try:
+        for medium in Media.objects.all():
+            if medium.slug not in MEDIA_CLASSES_DICT.keys():
+                raise APIException("".join([medium.name, " plugin is not registered in MEDIA_PLUGINS"]))
```

Comment:
While we're at it, we can improve the readability:
```suggestion
                raise APIException(f"{medium.name} plugin is not registered in MEDIA_PLUGINS")
```